data/sql/updates/pending_db_world/rev_1587562111390803800.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1587562111390803800');
+
+DELETE FROM `acore_string` WHERE entry IN (30081,30082);
+INSERT INTO `acore_string` (`entry`, `content_default`) VALUES
+(30081, 'You cannot share quests while in a Battleground'),
```

Comment:
normal that there is no final "." ? on both strings

Reply:
Why I dont add it is because personally I think a dot in a one line in a notification looks weird

Reply:
But it can always be altered if it is preferred with a dot, I'm down to change anything ^^

Reply:
Changed my mind, (.) was better

Reply:
xD